scripts/qa_crowbarsetup.sh

```diff
@@ -1547,9 +1547,20 @@ function jsonice()
     (echo -n '{}'; cat -) | sed -e 's/^{}\s*{/{/' | safely python -mjson.tool
 }
 
+function crowbar_any_status()
```

Comment:
MINOR: I would rather call it `crowbar_api_status` but I think it is a matter of taste